src/sparse/node.rs

```diff
@@ -43,6 +61,16 @@ impl Node {
         node
     }
 
+    pub fn height(&self) -> u32 {
+        let bytes = self.bytes_height();
+        u32::from_be_bytes(bytes.try_into().unwrap())
+    }
+
+    pub fn set_height(&mut self, height: u32) {
+        let bytes = height.to_be_bytes();
+        self.set_bytes_height(&bytes)
+    }
+
```

Comment:
Why is big endian used here, instead of native endian `to_ne_bytes`? I'm not saying that this is a mistake, but simply trying to understand

Reply:
There's no particular reason I used one over the other, but native endian would be good. There's no reason to prescribe big or little endian here, so I can defer to the native endian implementation.

Reply:
Ultimately, I think I would prefer to keep it as big endian. One concern with using native endian is that the storage where the buffer is persisted would not be portable across different endian platforms. It's conceivable that one might want to write the buffer using a big endian OS, but later read it using a little endian OS. Sticking to a single endian ensures that we have portability.

Reply:
If that is a concern, then sticking to a fixed endianness is definitely the right choice.